src/game/editor/mapitems/layer_game.cpp

```diff
@@ -38,8 +38,8 @@ void CLayerGame::SetTile(int x, int y, CTile Tile)
 		}
 		CTile nohook = {TILE_NOHOOK};
```

Comment:
```suggestion
		CTile NoHook = {TILE_NOHOOK};
```
This isn't caught by the CI?